README.md

```diff
@@ -4,10 +4,10 @@
 
 ## Installation
 
-Install with [npm](https://npmjs.com) globally:
+Install with [npm](https://npmjs.com) globally from the top level `device-os/user/tests/integration/` directory:
```

Comment:
It doesn't matter from which directory to install the runner tool itself as it's installed globally. Running `npm install` (without other arguments) from the `integration` directory is required to install JavaScript dependencies of the Device OS tests.

Reply:
Oh you know what happened... I installed it, then updated npm with nvm to the latest v12, then it was missing so I had to reinstall it :D   I'll change this back.

Reply:
Updated.